test/conformance/testdata/conformance.yaml

```diff
@@ -659,7 +659,7 @@
     API Server using the current Aggregator [Conformance]'
   description: Ensure that the sample-apiserver code from 1.17 and compiled against
     1.17 will work on the current Aggregator/API-Server.
-  release: v1.17
+  release: v1.17, v1.21
```

Comment:
what does this do?

Reply:
It just documents that the test was first introduced to conformance in 1.17 but that there was a change in behavior in 1.21